js/app.js

```diff
@@ -1,74 +1,101 @@
 require('@uswds/uswds');
+import AnchorJS from 'anchor-js';
+const anchors = new AnchorJS();
 
-const sideNavParents = document.querySelectorAll(".usa-sidenav__item--parent-item");
-const sideNavChild = document.querySelectorAll(".usa-sidenav__sublist--middle-generation");
-const subLists = document.querySelectorAll("ul.usa-sidenav__sublist");
-const middleSubList = document.querySelectorAll(".usa-sidenav__sublist ul");
 
-if (!!anchors) {
+function ready(callback) {
+  if (document.readyState !== 'loading') { // in case the document is already rendered
+    callback();
+  } else if (document.addEventListener) { // modern browsers
+    document.addEventListener('DOMContentLoaded', callback);
+  } else { // IE <= 8
```

Comment:
its ok to leave this in, but we don't have to support anything older than IE 11 going forward